### ISSUE TITLE
Add support for directory mods (Uncompressed .pk3)

### ISSUE THREAD

**Issue by braveatnight** [flegald/GZDeck#13]
Some mods that I use are uncompressed (Mainly [DoomRPG-Rebalance](https://github.com/WNC12k/DoomRPG-Rebalance)) and compressing them into .pk3 zip archives introduces some strange errors. GZDeck doesn't seem to know that mods can be in a directory (If it does and I somehow missed it you can let me know!). If you can add support for directory mods that would be fantastic because then I wouldn't need to compress them for game mode compatibility

**Comment by flegald** [comment]
Hi! Do you mean having like mods in a zip file?

**Comment by braveatnight** [comment]
Yeah I mean loading folders, Doom RPG distributes it's mods in folders that aren't zip or pk3 (did you know a pk3 just a zip with a different extension?) so if I want to use them with DeckGZ I need to compress them but that introduces some errors so if I could load folders that would be fantastic.
